site/content/docs/5.3/utilities/z-index.md

```diff
@@ -16,11 +16,11 @@ We call these "low-level" `z-index` utilities because of their default values of
 {{< /callout >}}
 
 {{< example class="bd-example-zindex-levels position-relative" >}}
-<div class="z-3 position-absolute p-5 rounded-3"></div>
-<div class="z-2 position-absolute p-5 rounded-3"></div>
-<div class="z-1 position-absolute p-5 rounded-3"></div>
-<div class="z-0 position-absolute p-5 rounded-3"></div>
-<div class="z-n1 position-absolute p-5 rounded-3"></div>
+<div class="z-3 position-absolute p-5 rounded-3"><span>z-3</span></div>
```

Comment:
I like the idea to display the classes here. However there's an issue with the rendering on dark mode where there's not enough contrast.

![Screenshot 2023-01-04 at 15 59 16](https://user-images.githubusercontent.com/17381666/210583606-b8cd4597-262c-4ef6-9017-1031c1542c02.png)

Reply:
Do you think I should force black color ?

Reply:
For consistency I'd use the same set of colors as in https://getbootstrap.com/docs/5.3/layout/grid/ and https://getbootstrap.com/docs/5.3/layout/gutters/ in light/dark mode.
But let's wait for @mdo thoughts before doing any modifications.

Reply:
I think I originally had these as translucent elements, but the feedback was that it make things a little less clear. I just use the inherited body color in those grid and gutter examples. Open to doing that again though to make this work better. Could also use some `.text-dark` so that it never changes color in dark mode.